spec/support/page_objects/publish/allocations/request/who_are_you_requesting_a_course_for_page.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative "../../../sections/radio_button"
+
+module PageObjects
+  module Publish
+    module Allocations
+      module Request
+        class WhoAreYouRequestingACourseForPage < PageObjects::Base
+          set_url "/publish/organisations/{provider_code}/{recruitment_cycle_year}/allocations/request"
+
+          element :header, "h1"
+
+          sections :radio_buttons, Sections::RadioButton, ".govuk-radios__item"
+
+          element :continue, ".govuk-button"
+
+          element :training_provider_search, "#training-provider-query-field"
+
+          def providers
+            @providers ||=
+              radio_buttons.reject do |x| x == find_an_organisation_not_listed_above end
+          end
+
+          def find_an_organisation_not_listed_above
+            @find_an_organisation_not_listed_above ||= radio_buttons.find do |x| x.text == "Find an organisation not listed above" end
```

Comment:
```suggestion
@find_an_organisation_not_listed_above ||= radio_buttons.find { |x| x.text == "Find an organisation not listed above" }
```

Reply:
Fixed, we really should invest in our linter through.